### PR TITLE
Feature/orams edit profile

### PR DESCRIPTION
**Pull request by marcoalfonso**

### CODE REVIEW

File: apps/orams/components/AwardsForm/AwardsForm.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+import Layout from 'shared/Layout'
+
+import BaseForm from 'shared/form/BaseForm'
+import SubmitForm from 'shared/form/SubmitForm'
+import ErrorBox from 'shared/form/ErrorBox'
+import MultiInput from 'shared/form/MultiInput.js'
+import formProps from 'shared/formPropsSelector'
+import { loadProfile } from 'orams/actions/profileActions'
+
+class AwardsForm extends BaseForm {
```
Comment (matt-sm) [inline apps/orams/components/AwardsForm/AwardsForm.js]: suggestion, not saying we do it, but these forms be stateless

File: apps/orams/components/AwardsForm/AwardsForm.js [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+  static propTypes = {
+    action: PropTypes.string,
```
Comment (matt-sm) [inline apps/orams/components/AwardsForm/AwardsForm.js]: missing some propTypes

File: apps/orams/components/AwardsForm/index.js [hunk 3]
```diff
@@ -0,0 +1,2 @@
+import AwardsForm from './AwardsForm'
```
Comment (matt-sm) [inline apps/orams/components/AwardsForm/index.js]: why do we need this?

File: apps/orams/components/AwardsForm/AwardsForm.js (hunk 2 above)
Comment (marcoalfonso) [reply apps/orams/components/AwardsForm/AwardsForm.js]: Well are we going to do things 'properly' with style guide or should we just 'quick' merge to show progress? There is a lot of code that can be refactored, but then again it is 57 files :/

File: apps/orams/components/AwardsForm/index.js (hunk 3 above)
Comment (marcoalfonso) [reply apps/orams/components/AwardsForm/index.js]: I didn't optimize the code. I basically just did as fast as possible the feature, which seems to be the 'ethos' of what we discussed.

File: apps/orams/components/AwardsForm/AwardsForm.js (hunk 1 above)
Comment (marcoalfonso) [reply apps/orams/components/AwardsForm/AwardsForm.js]: I agree, but should we spend time doing this?

File: apps/orams/pages/EditProfilePage.js [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+import BusinessDetailsForm from 'orams/components/BusinessDetailsForm'
+import BusinessInfoForm from 'orams/components/BusinessInfoForm'
+import YourInfoForm from 'orams/components/YourInfoForm'
+import ToolsForm from 'orams/components/ToolsForm'
+import AwardsForm from 'orams/components/AwardsForm'
+import SubmitStepForm from 'orams/components/Submit'
+import FinishProfile from 'orams/components/FinishProfile'
+import LocalNav from 'shared/LocalNav'
+
+class EditProfilePage extends Component {
```
Comment (matt-sm) [inline apps/orams/pages/EditProfilePage.js]: we may need to refactor this page when we do profile view.  Maybe:
`/profile` - view profile
'/profile/edit` - edit business details
`/profile/edit/business-info`
but we can refactor when we do view profile

**Comment by matt-sm** [comment]
conflicts?

File: apps/orams/components/AwardsForm/AwardsForm.js (hunk 2 above)
Comment (matt-sm) [reply apps/orams/components/AwardsForm/AwardsForm.js]: I agree its probably more important to show progress right now

File: apps/orams/components/BusinessInfoForm/BusinessInfoForm.js [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+class BusinessInfoForm extends BaseForm {
+  static propTypes = {
+    action: PropTypes.string,
+    csrf_token: PropTypes.string,
+    form: PropTypes.object.isRequired,
+    returnLink: PropTypes.string
+  }
+
+  componentDidMount() {
+    this.props.loadProfileData(this.props.form.model)
```
Comment (Blackfry) [inline apps/orams/components/BusinessInfoForm/BusinessInfoForm.js]: when I've used the load initial data action, it was in `componentWillMount`, wonder if it makes a difference?

**Pull request merged**